src/front-end/typescript/lib/pages/organization/edit/tab/team.tsx

```diff
@@ -245,9 +276,12 @@ function membersTableBodyRows(props: ComponentViewProps<State, Msg>): Table.Body
   const { state, dispatch } = props;
   const isAddTeamMembersLoading = state.addTeamMembersLoading > 0;
   const isRemoveTeamMemberLoading = !!state.removeTeamMemberLoading;
-  const isLoading = isAddTeamMembersLoading || isRemoveTeamMemberLoading;
+  const isApproveAffiliationLoading = !!state.approveAffiliationLoading;
+
+  const isLoading = isAddTeamMembersLoading || isRemoveTeamMemberLoading || isApproveAffiliationLoading;
   return state.affiliations.map(m => {
     const isMemberLoading = state.removeTeamMemberLoading === m.id;
+    const isApproveLoading = state.approveAffiliationLoading === m.id;
```

Comment:
Why is it called `approveAffiliationLoading` is it checking if it's currently loading? and to what end?

Reply:
it deactivates the buttons (and greys them out) while an async process occurs.  This prevents unwanted effects like double clicking on buttons or worse multiple api calls.